src/main/kotlin/pt/up/fe/ni/website/backend/model/Post.kt

```diff
@@ -0,0 +1,14 @@
+package pt.up.fe.ni.website.backend.model
+
+import java.util.*
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+
+@Entity
+class Post (
+    var description: String,
+    var link: String,
```

Comment:
I don't have a clear idea of what the Post should be, therefore I just thought it would be like the social media posts but with the link to the resource mentioned, for example, a post about TTS could have the link _ni.fe.up.pt/tts/planner_ or link to the page on the website dedicated to the project.

Reply:
I get what you're thinking. As for the model fields, this is my opinion:
- I understand the posts are usually about something (like a project or an event) but I don't think that'll always be the case. I think the links should simply be made with an `href` (assuming the post's body will be HTML, it can be something else but the concept still applies)
- I think `description` should be changed to `body` and we should add a `title` field. That makes more sense to me for a post.
- While we're at it, we could change `date` to `publishDate`, just to be more clear
- I would consider adding a template (a string called `templatePath` or something like that)

What do you think?